pkg/operator2/starter.go

```diff
@@ -195,51 +254,88 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 
 	ingressStateController := ingressstate.NewIngressStateController(
 		kubeInformersNamespaced,
-		kubeClient.CoreV1(),
-		kubeClient.CoreV1(),
-		operatorClient,
+		operatorCtx.kubeClient.CoreV1(),
+		operatorCtx.kubeClient.CoreV1(),
+		operatorCtx.operatorClient,
 		"openshift-authentication",
 		controllerContext.EventRecorder)
 
 	// TODO remove this controller once we support Removed
-	managementStateController := management.NewOperatorManagementStateController("authentication", operatorClient, controllerContext.EventRecorder)
+	managementStateController := management.NewOperatorManagementStateController("authentication", operatorCtx.operatorClient, controllerContext.EventRecorder)
 	management.SetOperatorNotRemovable()
 	// TODO move to config observers
 	// configobserver.NewConfigObserver(...)
 
-	for _, informer := range []interface {
-		Start(stopCh <-chan struct{})
-	}{
-		kubeInformersNamespaced,
-		kubeSystemNamespaceInformers,
-		authOperatorConfigInformers,
-		routeInformersNamespaced,
-		configInformers,
-		resourceSyncerInformers,
-	} {
-		informer.Start(ctx.Done())
-	}
+	operatorCtx.informersToRunFunc = append(operatorCtx.informersToRunFunc, kubeInformersNamespaced.Start)
```

Comment:
make it one append, with one element per line

Reply:
same for controllersToRunFunc

Reply:
it doesn't look pretty - check it out